### PR TITLE
Added Cart

### PR DESCRIPTION
**Pull request by syed-shaida-hussain**
please review this PR for Ecommerce Cart.
Screenshot (44)

### CODE REVIEW

File: Cart/cart.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    <link rel="stylesheet" href="/Cart/cart.css">
+    <link rel="stylesheet" href="https://garnish-ui.netlify.app/styles.css">
+</head>
+
+<body>
+
+    <h1 class="heading-text center-text">My Cart(1)</h1>
+
+    <hr>
+
```
Comment (arjunbharti) [inline Cart/cart.html]: Remove extra spacing

File: Cart/cart.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                <button class="remove-cart-btn">Remove from cart</button>
+                <button class="add-to-wishlist-btn">Add to wishlist</button>
+            </div>
+        </div>
+
+        <section class="price-details">
+            <p class="margin-top-bottom">PRICE DETAILS</p>
+
+            <hr />
+
```
Comment (arjunbharti) [inline Cart/cart.html]: Extra spacing here too

File: Cart/cart.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+                <p>-Rs 1999</p>
+            </div>
+
+            <div class="flex margin-top-bottom">
+                <p>Delivery charges</p>
+                <p>+Rs 99</p>
+            </div>
+
+            <hr />
+
```
Comment (arjunbharti) [inline Cart/cart.html]: Remove extra spacing

File: Cart/cart.html [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+            <div class="flex margin-top-bottom">
+                <p>Total Amount</p>
+                <p>-Rs 3099</p>
+            </div>
+
+            <hr />
+
```
Comment (arjunbharti) [inline Cart/cart.html]: Remove extra spacing

**Review by arjunbharti — APPROVED** [review]
Remove extra spacing in your code to make in more clean. Everything else is fine.

**Comment by poojakelkar** [comment]
Add the total amount in bold. Everything looks good to merge

**Pull request merged**